src/components/Elements/FormInputs/SelectInput.tsx

```diff
@@ -95,17 +102,19 @@ export const SelectInput = ({
         xl={3}
         xs={12}
       >
-        <Button
-          sx={{
-            maxHeight: '40px',
-          }}
-          onClick={() => {
-            openModal(modalContent);
-          }}
-          variant="contained"
-        >
-          Create
-        </Button>
+        {createButton && (
```

Comment:
[Podobno lepiej nie używać takiej formy](https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx), bo jest ryzyko że coś pójdzie nie tak jak się zaplanowało. Ale to tylko na przyszłość tutaj nic złego się nie powinno wydarzyć